### PR TITLE
Agnihotriketan/09122022

### PR DESCRIPTION
**Pull request by agnihotriketan**

### CODE REVIEW

File: csharp/backtracking/TreeMaze.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NeetCodeTester.backtracking
```
Comment (neetcode-gh) [inline csharp/backtracking/TreeMaze.cs]: Should we remove the namespace here and in TreeNode.cs to be consistent with the others?

**Review by neetcode-gh — CHANGES_REQUESTED** [review]
LGTM - just one comment, thanks.

**Comment by agnihotriketan** [comment]
Ok will do

**Comment by agnihotriketan** [comment]
done

**Comment by neetcode-gh** [comment]
Thank you, looks good!

**Pull request merged**